index.html

```diff
@@ -40,6 +40,7 @@
 
         <hr>
 
+        <input type="search" class="light-table-filter" data-table="sortable" placeholder="Filter" size="100">
```

Comment:
Use `style="width: 100%"` instead of size="100", which just means 100 characters